### ISSUE TITLE
Unable to use JSON parser

### ISSUE THREAD

**Issue by NathanFlegg** [happyleavesaoc/aoc-mgz#83]
The code snippet for the JSON parser doesn't seem to be working. I've got the full parser working but one of the latest DE test files (provided in this repo) is not working with the JSON parser. Here is the error I get:

```
/<my_absolute_path>/bin/python /<my_absolute_path>/test.py
Traceback (most recent call last):
  File "/<my_absolute_path>/lib/python3.9/site-packages/mgz/fast/header.py", line 461, in parse
    de = parse_de(header, version, save)
  File "/<my_absolute_path>/lib/python3.9/site-packages/mgz/fast/header.py", line 338, in parse_de
    de_string(data)
  File "/<my_absolute_path>/lib/python3.9/site-packages/mgz/fast/header.py", line 47, in de_string
    assert data.read(2) == b'\x60\x0a'
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/<my_absolute_path>/test.py", line 31, in <module>
    match = parse_match(h)
  File "/<my_absolute_path>/lib/python3.9/site-packages/mgz/model/__init__.py", line 93, in parse_match
    data = parse(handle)
  File "/<my_absolute_path>/lib/python3.9/site-packages/mgz/fast/header.py", line 469, in parse
    raise RuntimeError("could not parse")
RuntimeError: could not parse
```
For this file:
[de-26.16.zip](https://github.com/happyleavesaoc/aoc-mgz/files/8859121/de-26.16.zip)

Is the JSON parser not working yet for this version of DE recordings?

**Comment by happyleavesaoc** [comment]
Hi, the JSON output relies on the `fast` parser, which currently is not supported on the main branch. You can check out the `fix-fast` branch if you want to test the fix.

I'm testing the fix myself right now and will be merging it soon-ish.

**Comment by happyleavesaoc** [comment]
I pushed a commit that should fix it. I think this was some sort of incompatibility between python versions.

**Comment by NathanFlegg** [comment]
Amazing, it seems to be _pretty much_ working now. I've noticed in the replays though that under the "actions" key for these JSON files, all the positions are wrong, e.g. 

```
    {
      "timestamp": "0:00:01.976000",
      "type": "BUILD",
      "payload": {
        "building_id": 70,
        "building": "House"
      },
      "player": 8,
      "position": -7826587719021397205
    }
```

I'd expect it to be something like x = 61, y = 80 or something, not a random integer.
Any idea why this is and whether it's happening for you too? File here: [AgeIIDE_Replay_161780038.zip](https://github.com/happyleavesaoc/aoc-mgz/files/8871087/AgeIIDE_Replay_161780038.zip)

I used the JSON snippet you've provided (with a couple of file renaming additions but nothing that should affect the output. It seems to be a fast-parser only problem (or maybe JSON only?) because the full parser is producing all of the positions for moves that it should be.
